CIS560-RecipeManager/CIS560-RecipeManager/Pantry/partialQuery_Ingredient.cs

```diff
@@ -0,0 +1,118 @@
+﻿/**
+ * partialQuery_Ingredient.cs
+ * partial Query class; specific to Adding Ingredients to the database
+ * 
+ * author: Project Team 1 - CIS 560 KSU Fall 2018
+ * edited by: David K. Hwang
+ * 
+ */
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data;
+using System.Data.SqlClient;
+using System.Transactions;
+
+namespace CIS560_RecipeManager.Pantry
+{
+    public partial class Query : IQuery
+    {
+
+        public TransactionScope Transaction;
+
+        public SqlConnection Connection;
+
+        public Query()
+        {
+            Transaction = new TransactionScope();
+            Connection = new SqlConnection(Properties.Settings.Default.RecipeDatabaseConnectionString);
+        }
+
+        /// <summary>
+        /// Opens the connection to the database
+        /// </summary>
+        public void OpenConnection()
```

Comment:
Maybe we could have a partial Query class that handles the connection - as I think we want one connection for the entire application, but separate transactions for each query.

Reply:
Actually, I take this back.  It looks like we do want to open and close one connection per transaction.  It looks like opening and closing a SQLConnection isn't actually very expensive because ADO.NET does connection pooling behind the scenes (https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql-server-connection-pooling).  I think we should just use a using statement for each query method (as you were originally thinking).  Sorry about the confusion!